docs/rancher/import-existing-vm.md

```diff
@@ -0,0 +1,143 @@
+---
+sidebar_position: 8
+sidebar_label: Import Existing VM
+title: "Import Existing VM"
```

Comment:
Should this be "Import Existing Cluster built on Harvester VM"?

Reply:
Yes, that is better. Updated it!